portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="addMyComments()">
```

Comment:
Doesn't have to be done in this PR but does it still work if you move `onload="addMyComments()"` to `<div id="comments-container">`?

Reply:
Yes, 
I moved it to line 20: 
`<script>addMyComments();</script>`